config/database.yml

```diff
@@ -22,8 +22,14 @@ default: &default
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
 
 development:
-  <<: *default
-  database: ror_social_scaffold_development
+  <<: *default  
+  adapter: postgresql
+  username: postgres
+  password: 12345
+  host: localhost
+  port: 5432
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
```

Comment:
- [ ] **Database Configuration:** I got multiple errors while trying to migrate your database due to the fact that you've set most of the pg configuration for development. let's make it work in the test environment and the production environment
```yml
default: &default
  adapter: postgresql
  username: postgres
  password: 12345
  encoding: unicode
  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
  host: localhost
  port: 5432

development:
  <<: *default
  database: ror_social_scaffold_development
```